src/background/settings.js

```diff
@@ -23,4 +27,4 @@ function normalizeSettings(settings) {
 /**
  * Returns the configuration object for the Chrome extension
  */
-export default normalizeSettings(/** @type {any} */ (window).EXTENSION_CONFIG);
+export default normalizeSettings(settings);
```

Comment:
Interestingly, this could be one liner: `export default { ...settings, apiUrl: settings.apiUrl.replace(/\/$/, '') };`

Reply:
Indeed. I've made this change with one slight modification - the object needs to be cast to the `Settings` type to avoid errors in other modules about optional settings which happen not to exist in whatever settings file you most recently used.